calibrations/mbd/prorun_mbdcal.sh

```
@@ -162,15 +161,14 @@ mkdir -p ${caldir}
-#then
-#  ./cups.py -r ${runno} -s ${segment} -d ${outbase} message "Stage in pass0 from ${pass0dir}"
-#  for calib in mbd_shape.calib mbd_sherr.calib mbd_timecorr.calib mbd_slewcorr.calib mbd_tt_t0.calib mbd_tq_t0.calib mbd_pileup.calib
-#  do
-#    echo Stagein ${pass0dir}/${calib} to ${caldir}
-#    cp -p ${pass0dir}/${calib} ${caldir}/
-#  done
-#fi
+if [[ ! -z ${pass0dir} ]]
+then
+  #./cups.py -r ${runno} -s ${segment} -d ${outbase} message "Stage in pass0 from ${pass0dir}"
+  root.exe -b -q DumpMbdCalibs.C\(${runno}\)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)